records_mover/records/prep_and_load.py

```diff
@@ -25,9 +26,16 @@ def prep_and_load(tbl: TargetTableDetails,
         #
         #  Cannot COPY into nonexistent table
         driver = tbl.db_driver(db)
+        loader = driver.loader()
+        # TODO: Can this exception be passed in so we can skip this logic here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker